test/actor.js

```diff
@@ -703,7 +703,7 @@ describe('Apify.setValue()', () => {
                 promise: Promise,
                 key,
                 body: JSON.stringify(value, null, 2),
-                contentType: 'application/json',
+                contentType: 'application/json; charset=utf-8',
```

Comment:
Maybe one thing - we should also test the situation where contentType has no charset